### PR TITLE
Lexer Partnership Docs

### PR DESCRIPTION
**Pull request by KellieHawks**
@13tales
Updated submitted documentation to align with other Braze documentation.
Your documentation will be added to the site once we get the go-ahead from our partnership team.
Please feel free to reach out if you have any issues.

Thanks,
The Braze Documentation Team

In reference to this closed PR https://github.com/braze-inc/braze-docs/pull/3159

### CODE REVIEW

**Comment by KellieHawks** [comment]
@cla-bot check

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/lexer.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+### Step 1: Create activation
+
+Lexer Activate will automatically update your Braze profiles, adding or removing attributes as customers enter and leave your segment.
+
+1. In Lexer, in **Lexer Activations**, click **ACTIVATE NEW AUDIENCE**.
+2. Select the appropriate Braze activation for this campaign.
+3. Add your Segment.
+4. Update your audience name; this will become your attribute value in Braze.
+5. This is the custom attribute we’ll be updating in Braze. Contact [Lexer support](support@lexer.io) to update.
+6. Check the appropriate list action - in most cases, you’ll want to maintain your list.
```
Comment (bre-fitzgerald) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/lexer.md:59]:
```suggestion
6. Check the appropriate list action—in most cases, you’ll want to maintain your list.
```

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/lexer.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+7. Agree to terms, and click **SEND AUDIENCE**.
```
Comment (bre-fitzgerald) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/lexer.md:60]:
```suggestion
7. Review the terms and conditions, and click **SEND AUDIENCE**.
```

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/lexer.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+## Sending segments to Braze
+
+### Step 1: Create activation
+
+Lexer Activate will automatically update your Braze profiles, adding or removing attributes as customers enter and leave your segment.
+
+1. In Lexer, in **Lexer Activations**, click **ACTIVATE NEW AUDIENCE**.
+2. Select the appropriate Braze activation for this campaign.
+3. Add your Segment.
```
Comment (bre-fitzgerald) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/lexer.md:56]: It looks like this is the Braze segment and not the company Segment, so I believe it shouldn't be capitalized
```suggestion
3. Add your segment.
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Looks good! Few comments for your review @KellieHawks

**Pull request merged**